### PR TITLE
#4118 Add KmsKeyId Support to secretsmanager

### PR DESCRIPTION
**Pull request by glungley**

### CODE REVIEW

File: moto/secretsmanager/models.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def __init__(
         secret_binary=None,
         description=None,
         tags=[],
+        KmsKeyId=None,
```
Comment (bblommers) [inline moto/secretsmanager/models.py:64]: Variables should be snake-case (`kms_key_id`) (everywhere)

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @glungley, thanks for the PR!
Looks like the linter is failing, you might have to run black to get the CI to pass:
```
pip install black==19.10b0
make format
```

File: moto/secretsmanager/models.py (hunk 1 above)
Comment (glungley) [reply moto/secretsmanager/models.py]: Addressed, does a module somewhere else address that the boto API has the values in camel-case?

**Comment by bblommers** [comment]
Hi @glungley, make sure to use the right version of black (`19.10b0`), as other versions will result in different formatting

File: moto/secretsmanager/models.py (hunk 1 above)
Comment (bblommers) [reply moto/secretsmanager/models.py]: Ah, I missed that - no, we'd have to update the `response.py` file as well to read that param:
https://github.com/spulec/moto/blob/master/moto/secretsmanager/responses.py#L43

That goes for both create_ and update_secret.

Comment (glungley) [reply moto/secretsmanager/models.py]: Got the unit tests running locally, so have fixed that now. Thanks for the help btw

**Review by bblommers — APPROVED** [review]
LGTM - thanks again @glungley!

**Pull request merged**